WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1277,21 +1286,30 @@ private extension EditableOrderViewModel {
         }
     }
 
-    /// Summary of coupon lines
+    /// Coupon Line view models
     /// - Parameter couponLines: order's coupon lines
-    /// - Returns: Coupon codes comma separated to display as a summary
-    func summarizeCoupons(from couponLines: [OrderCouponLine]) -> String? {
-        guard couponLines.isNotEmpty else {
-            return nil
+    /// - Returns: View models for the coupon lines, including the view model for the details screen in case it's navigated to
+    ///
+    func couponLineViewModels(from couponLines: [OrderCouponLine]) -> [CouponLineViewModel] {
+        couponLines.map {
+            CouponLineViewModel(title: String.localizedStringWithFormat(Localization.CouponSummary.singular, $0.code),
+                          discount: "-" + (currencyFormatter.formatAmount($0.discount) ?? "0.00"),
```

Comment:
Could hardcoding the `-` and `0.00` value here cause any issue in other currencies/locales when rendering? I'm thinking RTL languages since the formatter only deals with the amount:

```
(lldb) po currencyFormatter.formatAmount("10")
▿ Optional<String>
  - some : "$10.00"
```

Reply:
[This](https://github.com/woocommerce/woocommerce-ios/blob/7364cac6e447387836f1cb0424a22e91d6673842/WooCommerce/Classes/ViewRelated/Orders/Order%20Creation/EditableOrderViewModel.swift#L727) was there before, so far it didn't give any problem. But I will keep an eye in case it happens. Thanks for your comment